components/ImaSdkTask.brs

```diff
@@ -205,14 +230,15 @@ sub setupVideoPlayer()
         ? "TRUE[X] >>> ImaSdkTask::adBreakStarted(adBreakInfo=";adBreakInfo;")"
         m.top.currentAdBreak = adBreakInfo
         m.top.adPlaying = true
-        m.top.video.enableUi = false
-        m.top.video.enableTrickPlay = false
     end function
 
     m.player.adBreakEnded = function(adBreakInfo as Object)
         ? "TRUE[X] >>> ImaSdkTask::adBreakEnded(adBreakInfo=";adBreakInfo;")"
         m.top.adPlaying = false
-        m.top.video.enableTrickPlay = true
+        if m.top.snapbackTime > m.top.video.position then
```

Comment:
I'm not sure I understand the snapbackTime logic?

Reply:
I think [the image](https://developers.google.com/interactive-media-ads/images/dai_snapback2.png) is the most helpful way to visualize it.

1. User is between ad breaks, let's say at time 10
2. User wants to seek to time 50, but there's an ad break at time 25
3. Snapback to 25 to show the ads
4. When ad break completes send user to their requested position, t=50

Is your previous comment suggesting that instead of step 4 we just let the stream continue playing directly after the ad break at t=25?